### DISCUSSION TITLE
Add a new function to eventCommands

### DISCUSSION THREAD

**Discussion by Snele** [Zrips/CMI#5443] [Feature Requests]
Add an event with appearance of the operator.
For example, I want the operator to be deopped when it was opped.
This will greatly help in additional server protection!

**Comment by dkalaxdk** [comment]
Why not just disallow people to be modded? 
This seems like it is more related to the permissions that you've granted to players, rather than a CMI event.

**Reply by Snele** [reply]
It's not that simple. I live in Russia, where our people really like to crash servers. Even with minimal permissions, some players find vulnerabilities that would seem absurd, but can give out an OP. Even having protection in some plugins to block the /op command doesn't really help.
And what I'm asking for can really save the server. I have never seen a plugin that would immediately deop the player when it was recently opped.
There are some plugins that have the auto-deop function ON LOGIN, but I need it immediately if possible!

**Reply by dkalaxdk** [reply]
Understood.

But it does not seem like a feature within the scope of CMI, but would rather be in the scope of an anti-cheat plugin in my opinion.
